subprojects/tooling-api/src/main/java/org/gradle/tooling/TestLauncher.java

```diff
@@ -82,6 +83,17 @@
      */
     TestLauncher withJvmTestMethods(String testClass, Iterable<String> methods);
 
+    /**
+     * Runs the tests in debug mode.
+     *
+     * @param port the target port where to wait for the debugger
```

Comment:
❌ The test doesn't wait (it's not run in server mode), it connects to an already running debugger. We should explain the exact behavior or just provide the equivalent JVM argument that we are using.

I'd probably also rename the method to something like `debugTests` to make it clear that this is not about debug options for the main daemon.

Reply:
👍